src/contexts/ConfigurationContext.tsx

```diff
@@ -497,10 +497,10 @@ export const ConfigContextProvider = ({ children }: Props) => {
       setRerank({
         isEnabled: isTrue(config_mmr) || isTrue(config_rerank),
         numResults: isTrue(config_mmr)
-          ? config_mmr_num_results
+          ? (config_mmr_num_results || 50)
```

Comment:
actually 0 is never a good value here, so I think per your comment "||" is the right choice. Correct?

Reply:
I think I'd still suggest `??`, because even though 0 is a poor choice, it's also the expected behavior if someone configures `config_mmr_num_results` at 0. If someone specifies 0 they will expect to see it behave as if it is set to 0, but then to see it behave as if it's set to 50 will be very surprising and will require digging into the code to understand.